okhttp/src/main/java/okhttp3/internal/http1/Http1ExchangeCodec.kt

```diff
@@ -65,18 +65,18 @@ class Http1ExchangeCodec(
   /** The client that configures this stream. May be null for HTTPS proxy tunnels. */
   private val client: OkHttpClient?,
   /** The connection that carries this stream. */
-  private val realConnection: RealConnection?,
+  override val connection: RealConnection,
   private val source: BufferedSource,
   private val sink: BufferedSink
 ) : ExchangeCodec {
   private var state = STATE_IDLE
   private var headerLimit = HEADER_LIMIT.toLong()
 
-  private fun Response.isChunked() =
-      "chunked".equals(header("Transfer-Encoding"), ignoreCase = true)
+  private val Response.isChunked: Boolean
+      get() = "chunked".equals(header("Transfer-Encoding"), ignoreCase = true)
```

Comment:
Separate from this review, but should this be 

```
headers("Transfer-Encoding").any { "chunked".equals(it, ignoreCase = true) }
```

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Transfer-Encoding

> // Several values can be listed, separated by a comma
> Transfer-Encoding: gzip, chunked

Reply:
Ooooh, yeah that’s tricky.